tasks/postgresql.yml

```diff
@@ -0,0 +1,65 @@
+---
+## PostgreSQL installation and configuration for sympa
+
+- name: check PostgreSQL installation
+  block: 
+  - name: Gather installed packages
+    package_facts:
+      manager: auto
+
+  - name: check if postgres is installed
```

Comment:
Please start with a capital Letter

Reply:
You mean "Check" instead of "check"? Ok, you're right. I let you resolve this conversation and other similar ones.